src/permutat.cc

```diff
@@ -909,6 +944,10 @@ static Obj CommPerm(Obj opL, Obj opR)
     degC = degL < degR ? degR : degL;
     com = NEW_PERM<Res>( degC );
 
+    if (degL == 0 || degR == 0) {
```

Comment:
As above, I'd move these before the `NEW_PERM` call.

Reply:
Yes, the ebeing after the new perm defeats most of the purpose. Should have been more careful.